client/my-sites/patterns/components/pattern-library/index.tsx

```diff
@@ -286,18 +286,23 @@ export const PatternLibrary = ( {
 				</div>
 
 				{ isHomePage && (
-					<CategoryGallery
-						title={ translate( 'Ship faster, ship more', {
-							comment:
-								'Heading text for a section in the Pattern Library with links to block pattern categories',
-							textOnly: true,
-						} ) }
-						description={ translate(
-							'Choose from a library of beautiful, functional design patterns to build exactly the pages you need—or your client needs—in no time.'
+					<>
+						<CategoryGallery
+							title={ translate( 'Build anything with patterns', {
+								comment:
+									'Heading text for a section in the Pattern Library with links to block pattern categories',
+								textOnly: true,
+							} ) }
+							description={ translate(
+								'Choose from a library of beautiful, functional design patterns to build exactly the pages you need—or your client needs—in no time.'
+							) }
+							categories={ categories }
+							patternTypeFilter={ PatternTypeFilter.REGULAR }
+						/>
+						{ isEnabled( 'readymade-templates/showcase' ) && (
+							<PatternsCopyPasteInfo theme="gray" />
```

Comment:
I think we can make these changes to prod as well, no need to check the flag.

Previously, the flag was only used because the position of the elements changed with the new RMT section, but now, elements are always placed in the same position in prod and with the flag.

Reply:
Updated